GUI/GUIListPanel.cpp

```diff
@@ -723,7 +727,9 @@ void GUIListPanel::ScrollBarScrolling(int mouseWheelChange) {
 	if (mouseWheelChange < 0) {
 		newValue = m_VertScroll->GetValue() - (mouseWheelChange * avgItemHeight);
 		int maxValue = GetStackHeight(lastItem) + GetItemHeight(lastItem) - m_VertScroll->GetPageSize();
-		newValue = std::clamp(newValue, maxValue, 0);
+		if (newValue > maxValue) {
```

Comment:
Why do this over the old clamp?

Reply:
The else statement below did not use a clamp - made it be more in line, imo more readable.
Could've just swapped 0 and maxValue but I stand by this.

Reply:
Fair enough!